build/constants.yaml

```diff
@@ -27,14 +27,14 @@
     python-gunicorn-custom-thread-num: PYTHON_GUNICORN_CUSTOM_THREAD_NUM
     nginx-conf-file: NGINX_CONF_FILE
   outputs:
-    - type: csharp
```

Comment:
@william-msft do you know why `constants.yaml` is _seemingly_ always cleaning up indentations in these automated PRs?

Reply:
Yes this is because `yq` and `YamlDotnet` have different spacing rules. 

To fix I've submitted this PR: https://github.com/microsoft/Oryx/pull/2216/files#diff-3ed663dcfb84882e18b67fce2ab60dcb67f0b57ac69932ca286dcea41b5d80a5R135-R136

It stopped fixing spacing after the parameters got changed.